libfwupdplugin/fu-device.c

```diff
@@ -168,6 +175,115 @@ fu_device_add_possible_plugin (FuDevice *self, const gchar *plugin)
 	g_ptr_array_add (priv->possible_plugins, g_strdup (plugin));
 }
 
+/**
+ * fu_device_retry_add_condition:
```

Comment:
I think you should rename this to `add_reset_func`

Reply:
Or adjust documentation for the other one to reference calling the conditions not reset_func

Reply:
It's not specifically a reset tho, it's just a way to recover the current action. How about `fu_device_retry_add_recovery` or something?

Reply:
Sure, that's better than condition.